src/jitlog/reader.lua

```diff
@@ -286,6 +301,8 @@ local function makereader(mixins)
     eventid = 0,
     actions = {},
     markers = {},
+    exits = 0,
+    gcexits = 0, -- sub count of trace exits forced by the gc state being in atomic or finalize
```

Comment:
I can't quite parse the comment.

Reply:
They are caused from traces that allocate objects. They call in to the GC from there JIT'ed code to do a bit of work but as an optimization it skips some costly object barriers need when the GC is in the atomic(all house keeping that has to be done at the end of the mark phase) or finalize(run object finalizers) phase.

Reply:
OK, can you try updating the comment? My problem is more that I can't parse the English at the moment.

Reply:
comment updated